src/Client/Connection.h

```diff
@@ -169,6 +170,10 @@ class Connection : public IServerConnection
     String default_database;
     String user;
     String password;
+    String proto_send_chunked;
+    String proto_recv_chunked;
+    String proto_send_chunked_srv;
+    String proto_recv_chunked_srv;
```

Comment:
Let's write a comment about these fields. What they mean and how they are used. It's not obvious from their names.